device/dhcp/servers/v6/dhcpv6server.yaml

```diff
@@ -0,0 +1,212 @@
+components:
+  schemas:
+    DhcpServer.V6:
+      description: >-
+        Configuration for emulated DHCPv6 Server.
+      type: object
+      x-status:
+        status: under_review
+      required: [name,ipv6_name,leases]
+      properties:
+        name:
+          x-include: ../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        ipv6_name:
+          description: >-
+            The unique name of the IPv6 on which DHCPv6 server will run.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv6/properties/name'
+          x-field-uid: 2
+        rapid_commit:
+          description: >-
+            If Rapid Commit is set, server responds to client initiated Rapid Commit two-message exchanges.
+          type: boolean
+          default: false
+          x-field-uid: 3
+        reconfigure_via_relay_agent:
+          description: >-
+            If the server does not have an address to which it can send the Reconfigure message directly to the client, the 
+            server uses a Relay-reply message to send the Reconfigure message to a relay agent that will relay the message to
+            the client.
+          type: boolean
+          default: false
+          x-field-uid: 4
+        primary_dns_server:
```

Comment:
primary_dns_server etc are optional paramters in the lease, so created optional object.

Reply:
updated